pydocx/openxml/wml/object.py

```diff
@@ -0,0 +1,19 @@
+# coding: utf-8
+from __future__ import (
+    absolute_import,
+    print_function,
+    unicode_literals,
+)
+
+from pydocx.models import XmlModel, XmlCollection
+from pydocx.openxml.vml.shape import Shape
+
+
+class Object(XmlModel):
```

Comment:
Based on my investigation, it looks like `w:object` refers to this structure:

https://msdn.microsoft.com/en-us/library/documentformat.openxml.wordprocessing.embeddedobject%28v=office.15%29.aspx

I think this should be `EmbeddedObject` and be in the `wordprocessing` module. Thoughts?